src/App.js

```diff
@@ -0,0 +1,34 @@
+import './styles/App.scss';
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import MainPage from './components/HomePage/MainPage';
+import Details from './components/DoctorsData/DetailsPage/Details';
+import Appointments from './components/AppointmentsData/AppointmentsPage';
+import LoginPage from './components/Authentication/LoginPage';
+import RegistrionPage from './components/Authentication/RegistrionPage';
+import SplashPage from './components/SplashPage/SplashPage';
+import AddItem from './components/DoctorsData/AddItem';
+import DeleteItem from './components/DoctorsData/DeleteItem';
+import AppointmentsDetails from './components/AppointmentsData/DetailsPage/Details';
+import AddAppointmentsForm from './components/AppointmentsData/AddAppointment';
+
+function App() {
```

Comment:
- [ ] The "All my appointments" page has not been styled according to the design guidelines.  Kindly add some styling so it looks beautiful and professional. 

- [ ] Instead of showing the "ID_doctor", please indicate the doctor's name, so your users know who they'll visit.

- [ ] Please, try to format the dates accordingly so they are human-readable.  There are npm packages that make this easy for JS, like "dateformat".

That way, the UX of your appointments page will be a lot more usable, beautiful, and professional. 💪